Earthfile

```diff
@@ -80,6 +80,7 @@ OSRELEASE:
     ARG HOME_URL
 
     # update OS-release file
+    RUN sed -i -n '/KAIROS_/!p' /etc/os-release
```

Comment:
I guess this removed every line maching `KAIROS_`, right? It's worth a comment tbh, I don't find `sed` syntax to be very intuitive.